src/EventListener/StorageListener.php

```diff
@@ -99,7 +99,11 @@ public function preHydrate(HydrationEvent $event)
 				foreach ($localeData as $key => $value) {
 					if (isset($contentType['fields'][$key]['type']) && $contentType['fields'][$key]['type'] !== 'repeater') {
 						$subject[$key] = is_array($value) ? json_encode($value) : $value;
-					}
+					}  elseif ( $key == 'templatefields') {
```

Comment:
there are two spaces before `elseif` and one before $key, I try to sorta stick to PSR2